### PR TITLE
Added SOP for CPDLC Functionality with vatACARS

### PR DESCRIPTION
**Pull request by tylerthetiletiler**
## Summary
Added SOP for CPDLC Functionality with vatACARS

## Changes

**Additions**:
- Adding CPDLC Controller Skills page inline with vatACARS release for use on the network
- Amended ENR pages to include CPDLC section

### CODE REVIEW

**Comment by tylerthetiletiler** [comment]
Please do not merge, I will do it once given the Ok from the vatACARS team :)

File: docs/controller-skills/cpdlc.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+| INL | YINL | Voice |
+| ISA | YISA | **CPDLC** |
+| KEN | YKEN | Voice |
+| TRT | YTRT | **CPDLC** |
+| ASP | YASP | **CPDLC** |
+| BIK | YBIK | Voice |
+| ELW | YELW | Voice |
+| HUO | YHUO | Voice |
+| OLW | YOLW | **CPDLC** |
+| PIY | YPIY | **CPDLC** |
```
Comment (alphadelta332) [inline docs/controller-skills/cpdlc.md:99]: Should be Voice (arrivals sector)

File: docs/enroute/Brisbane Centre/ARL.md [hunk 2]
```diff
@@ -15,6 +15,18 @@
...
+### CPDLC
+
+The Primary Communication Method for ARL is Voice.
+
+[CPDLC](../../controller-skills/cpdlc.md) may be used in lieu when applicable.
+
+The CPDLC Station Code is `YARL`.
+
+!!! tip
+        Even though ARL's Primary Communication Method is Voice, CPDLC should be used for Overfliers.
```
Comment (alphadelta332) [inline docs/enroute/Brisbane Centre/ARL.md:28]: I feel like 'may' is a better choice of wording than 'should', but if you prefer to encourage its use then no worries

File: docs/enroute/Melbourne Centre/PIY.md [hunk 3]
```diff
@@ -16,6 +16,15 @@
 | Geraldton† | Melbourne Centre | 134.200 | ML-GEL_CTR |
 
 † *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies)
+
+### CPDLC
+
+The Primary Communication Method for PIY is [CPDLC](../../controller-skills/cpdlc.md).
```
Comment (alphadelta332) [inline docs/enroute/Melbourne Centre/PIY.md:22]: Voice

**Review by alphadelta332 — CHANGES_REQUESTED** [review]

File: docs/controller-skills/cpdlc.md (hunk 1 above)
Comment (tylerthetiletiler) [reply docs/controller-skills/cpdlc.md:99]: OLW or PIY? or both

File: docs/enroute/Brisbane Centre/TRT.md [hunk 4]
```diff
@@ -12,6 +12,17 @@
 
 † *Non-standard positions* may only be used in accordance with [VATPAC Air Traffic Services Policy](https://vatpac.org/publications/policies)
 
+### CPDLC
+
+The Primary Communication Method for TRT is Voice.
```
Comment (alphadelta332) [inline docs/enroute/Brisbane Centre/TRT.md:17]: Sorry missed this on the first go. Table on the CPDLC page has CPDLC as primary. Honestly it's probably halfway in between, so up to you which one you go

**Review by alphadelta332 — CHANGES_REQUESTED** [review]

**Review by alphadelta332 — APPROVED** [review]

**Review by chrisg06 — APPROVED** [review]

File: docs/controller-skills/cpdlc.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+**CPDLC (Expect for PDC's) must not be used below `F245`.**
+
+After logging onto the network the controller should open the *Setup* window, fill in the relevent details and click *Save & Connect*
+
+<figure markdown>
+![vatACARS Setup](../controller-skills/img/vatacarslogon.png){ width="700" }
+</figure>
+
+**Station Code:** Must be the identifier of the airport which is being controlled.
```
Comment (zkgell) [inline docs/controller-skills/cpdlc.md:72]: Of the position being controlled?

**Review by zkgell — APPROVED** [review]

**Pull request merged**